From d05657df30d7c0e686eaed9561b1d61587d1b50f Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Thu, 7 Nov 2024 15:50:22 +0100
Subject: [PATCH] Makefile fix: don't execute precheck during installation

Due to mypy modifications, we can't build the sm RPM in Koji
without a recent pylint version. So the precheck target is only
executed in a github workflow now.

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 293e5faa..3748226c 100755
--- a/Makefile
+++ b/Makefile
@@ -127,7 +127,7 @@ precheck: build
 	echo "Precheck succeeded with no outstanding issues found."
 
 .PHONY: install
-install: precheck
+install: build
 	mkdir -p $(SM_STAGING)
 	$(call mkdir_clean,$(SM_STAGING))
 	mkdir -p $(SM_STAGING)$(SM_DEST)
